coldkeypub.txt

```diff
@@ -1 +0,0 @@
-0xf872b891311b7078f17833b7064b0e0105f1e74553b0310b0e7fe661d9860c56
```

Comment:
Maybe let's add coldkeypub.txt to the .gitignore list, it'll keep getting pushed up every time otherwise.